index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;200;300;400;500;700&family=Poppins:wght@100;200;300;400;500&family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet">
+  <link
+    rel="stylesheet"
+    href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css"
+    type="text/css"
+  />
+  <link href="styles.css" rel="stylesheet" />
+</head>
+<body>
+  <header>
+    <nav class="header">
+      <a href="#" class="logo">Kaneez Fatima</a>
+      <div class="topnav-right">
+      <a><i class="fa fa-bars cursor"></i></a>
+    </div>
+    </nav>
+  </header>
+  <section class="main-section img">
+    <div>
+      <h1>Hey There.</h1>
+        <h1>I'm Kaneez Fatima</h1>
+        <h2>I'am a Software Developer</h2>
+      </div>
```

Comment:
- [ ]  A single page can only contain one `h1` tag. This helps with Search Engine Optimization and makes your code look more semantic. So for the `hey there` and `Im Kaneez Fatima` you can put them in one `h1` tag and separate them with `br` break tag in order to skip to the next line.
- [ ] Kindly increase the font size of the `h1` tag. This is to ensure we replicate the design as much as possible. You can check out the screenshots below for reference.
- Yours
![image](https://user-images.githubusercontent.com/68167810/125122560-47738480-e0fe-11eb-806b-abf8ca207f69.png)
 template's
![image](https://user-images.githubusercontent.com/68167810/125122235-cae0a600-e0fd-11eb-806e-97f4a4376723.png)

Reply:
Well resolved